test/suites/smoke-test-dancelight/test-external-validators-rewards.ts

```diff
@@ -99,21 +99,24 @@ describeSuite({
             test: async () => {
                 const historyDepth = api.consts.externalValidatorsRewards.historyDepth;
 
+                console.log(historyDepth);
+
                 // Checkpoint A: current era index - historyDepth
                 const eraIndexCheckpointA =
                     (await api.query.externalValidators.activeEra()).unwrap().index.toNumber() -
                     historyDepth.toNumber();
                 // Checkpoint B: eraIndexCheckpointA + 1
                 const eraIndexCheckpointB = eraIndexCheckpointA + 1;
 
-                const validatorRewardCheckpointA =
-                    await api.query.externalValidatorsRewards.rewardPointsForEra(eraIndexCheckpointA);
-
-                const validatorRewardCheckpointB =
-                    await api.query.externalValidatorsRewards.rewardPointsForEra(eraIndexCheckpointB);
-
-                expect(validatorRewardCheckpointA.isEmpty).toBe(true);
-                expect(validatorRewardCheckpointB.isEmpty).toBe(false);
+                // The mapping only contains the keys that are in `externalValidatorsRewards`
+                const rewardMappingKeys = (await api.query.externalValidatorsRewards.rewardPointsForEra.keys()).map(
+                    (key) => {
+                        // Convert "1,020" into 1020
+                        return Number.parseInt(key.toHuman().toString().replace(",", ""));
```

Comment:
Why not just convert key to number instead of human to string and replace the commas:

`(key) => key.toNumber()`

Reply:
@timbrinded , there is no `toNumber` helper for key.

Reply:
jsut copied it from another place, will apply this change